File: packages/@monomind/monograph/__tests__/fixtures/sample.dart
import 'package:flutter/material.dart';

abstract class Animal {
  String get name;
  String speak();
}

class Dog extends Animal {
  final String name;

  Dog(this.name);

  @override
  String speak() {
    return 'Woof!';
  }
}

class Cat extends Animal {
  final String name;

  Cat(this.name);

  @override
  String speak() {
    return 'Meow!';
  }
}

enum Direction { north, south, east, west }

String helperFn(String s) {
  return s.toUpperCase();
}

File: packages/@monomind/monograph/__tests__/fixtures/sample.swift
import Foundation

protocol Greetable {
  func greet() -> String
}

struct Person: Greetable {
  var name: String

  func greet() -> String {
    return "Hello, \(name)"
  }
}

class Animal {
  var name: String

  init(name: String) {
    self.name = name
  }

  func speak() {
    print("\(name) speaks")
  }
}

class Dog: Animal {
  override func speak() {
    print("Woof!")
  }
}

enum Direction {
  case north
  case south
  case east
  case west
}

func helperFn() -> Int {
  return 42
}
